api/schema/types/ClientType.js

```diff
@@ -35,7 +35,10 @@ module.exports = gql`
     type Query {
         getClientById(id: Int!): Client
         getClients: [Client]
+        getActiveClients: [Client]
+        getInactiveClients: [Client]
         getActiveClientsCount: Int!
+        getInactiveClientsCount: Int!
```

Comment:
Suggestion:

An optimization here might look like this:

```
getClients(active: Boolean): [Client]
getTotalClients(active: Boolean): Int!
```

which might result in some LOC savings

Does not need to be changed now but an idea for #474